examples/rethinkdb/v1beta3/README.md

```diff
@@ -0,0 +1,106 @@
+RethinkDB Cluster on Kubernetes
+==============================
+
+Setting up a [rethinkdb](http://rethinkdb.com/) cluster on [kubernetes](http://kubernetes.io)
+
+**Features**
+
+ * Auto configuration cluster by querying info from k8s
+ * Simple
+
+Quick start
+-----------
+
+**Step 1**
+
+antmanler/rethinkdb will discover peer using endpoints provided by kubernetes_ro service,
```

Comment:
Should read "...will discover it's peers by using endpoints provided by the kubernetes_ro service..."